src/improvements/template/TransferOwnerTemplate.sol

```diff
@@ -0,0 +1,57 @@
+pragma solidity 0.8.15;
+
+import {SystemConfig} from "@eth-optimism-bedrock/src/L1/SystemConfig.sol";
+import {ProxyAdmin} from "@eth-optimism-bedrock/src/universal/ProxyAdmin.sol";
+
+import {MultisigTask} from "src/improvements/tasks/MultisigTask.sol";
+import {IAddressRegistry} from "src/improvements/IAddressRegistry.sol";
+import {AddressRegistry as Addresses} from "src/improvements/AddressRegistry.sol";
+
+/// @title TransferOwnerTemplate
+/// @notice Template contract for transferring ownership of the proxy admin
+contract TransferOwnerTemplate is MultisigTask {
+    /// @notice new owner address
+    address public newOwner;
+
+    /// @notice Returns the safe address string identifier
+    /// @return The string "SystemConfigOwner"
+    function safeAddressString() public pure override returns (string memory) {
+        return "SystemConfigOwner";
+    }
+
+    /// @notice Returns the storage write permissions required for this task
+    /// @return Array of storage write permissions, in this case, only the ProxyAdmin is returned
+    function _taskStorageWrites() internal pure virtual override returns (string[] memory) {
+        string[] memory storageWrites = new string[](1);
+        storageWrites[0] = "ProxyAdmin";
+        return storageWrites;
+    }
+
+    /// @notice Sets up the template with the new owner from a TOML file
+    /// @param taskConfigFilePath Path to the TOML configuration file
+    function _templateSetup(string memory taskConfigFilePath) internal override {
+        newOwner = abi.decode(vm.parseToml(vm.readFile(taskConfigFilePath), ".newOwner"), (address));
```

Comment:
nit, I would suggest using `stdToml` and doing `vm.readFile(taskConfigFilePath).readAddress(".newOwner")` instead of `abi.decode`, since it's more readable

Reply:
Made PR #700 to see if we want to use `stdToml`.

In short, we would use `stdToml` rarely, but we can extract the `abi.decode(vm.parseToml(...))` calls into their own functions, and increase readability and usability by a bit, if we don't mind having a bunch of one-liner functions that get called only once (i.e. more boilerplate).